client/src/components/RelatedItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const RelatedItem = (props) => {
+    // console.log(props.relatedItemData);
+    return (
+        <div>
+            <li className = "related-item">
+                <div><img src={props.relatedItemData.image} className="related-item-image"/></div>
+                <div><a href="#">{props.relatedItemData.itemName}</a></div>
+                <div>Average Stars: {props.relatedItemData.averageStarRating}</div>
+                <div>Number of Reviews: {props.relatedItemData.numberOfReviews}</div>
+                <span>
+                    <div>Price: {props.relatedItemData.price}</div>
+                    {/* <div>Avail on P? {props.relatedItemData.availableOnPrime}</div> */}
```

Comment:
My data has a `prime_eligible` boolean property that indicates whether the product is prime eligible/available or not, so you can uncomment this if that's the reason why you have it commented out.

Reply:
Thanks for the reminder, main reason I did that was because I am still looking for a suitable icon, similar to what they have on the actual site.